pkg/kube/inject/inject.go

```diff
@@ -918,6 +921,21 @@ func structToJSON(v interface{}) string {
 	return string(ba)
 }
 
+func protoToJSON(v proto.Message) string {
+	if v == nil {
+		return "{}"
+	}
+
+	m := jsonpb.Marshaler{}
```

Comment:
add OrigName here. are you sure this needs gogo/protobuf as opposed to golang/protobuf jsonpb?

Reply:
as long as both output `interval: "10s"` instead of `interval: { seconds: 10}`, I am good.